src/content/docs/accounts/accounts-billing/new-relic-one-user-management/tutorial-add-new-user-groups-roles-new-relic-one-user-model.mdx

```diff
@@ -11,117 +11,74 @@ This tutorial will walk you through some common procedures for managing users on
 
 This tutorial will walk you through how to:
 
-* Step 1: [Add/manage groups](#step-1-groups)
-* Step 2: [Add/manage roles](#step-2-roles)
-* Step 3: [Manage accounts](#step-3-roles)
-* Step 4: [Grant groups access to roles/accounts](#step-4-access-grants) (access grants)
-* Step 5: [Add/manage users](#step-5-users)
+* [View available accounts](#accounts)
+* [How to grant groups access to roles and accounts](#group-access) 
+* [How to create custom roles](#roles)
+* [How to add users to groups](#add-users)
 
-<Callout variant="tip">
-  The tutorial steps are also summarized in the [Organization and access UI](https://one.newrelic.com/launcher/organizations-ui.organization-launcher).
-</Callout>
-
-## Step 1: Add/manage groups [#step-1-groups]
+## View available accounts [#accounts]
 
-Groups are used to manage what users are able to do in New Relic. By default, most New Relic organizations have two groups: [**Admin** and **User**](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model#group-roles). And you can add your own custom groups.
-
-<Callout variant="tip">
-  You can also [import users automatically from your identity provider](/docs/accounts/accounts/automated-user-management/automated-user-provisioning-single-sign).
-</Callout>
+Understanding the accounts in your New Relic organization can be an important first step because when you grant groups access, you must choose what account to give the group access to.
 
-To view your organization’s groups: go to the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary#account-dropdown), click **Organizations and access**, and click **Groups**.
+To view the accounts in your organization: go to the [account dropdown](/docs/using-new-relic/welcome-new-relic/get-started/glossary#account-dropdown), click **Administration**, click **Organizations and access**, and then click **Accounts**.
 
-To create a group:
+## Grant groups access to roles and accounts [#group-access]
 
-1. Click **Add group** and then name the group.
-2. Select the [authentication domain](/docs/accounts/accounts-billing/new-relic-one-user-management/configure-authentication-domains-sso) the group should belong to, and click **Add group**.
+Groups are used to manage what your users are able to do in New Relic. By default, organizations on the New Relic One account/user structure have two available groups: [**Admin** and **User**](/docs/accounts/accounts-billing/new-relic-one-user-management/new-relic-one-user-model#group-roles). These default groups automatically have access to specific standard roles and they are assigned to the account in which they were initially added. If you're on Pro or Enterprise tier, you additionally have the ability to create new groups, new roles, and assign groups to other accounts in your organization. 
```

Comment:
"if you're on our Pro or Enterprise tier"

"you additionally have the ability to create new groups and new roles, and to assign groups to other accounts in your organization."